msteams-platform/get-started/glossary.md

```diff
@@ -80,7 +79,6 @@ Common terms and definitions used in Teams developer documentation.
 
 | Term | Definition |
 | --- | --- |
-| [Federated user](../apps-in-teams-meetings/meeting-app-extensibility.md#user-types-in-a-meeting) | A type of user in a Teams app meeting who is external and is invited to the meeting. This user has valid credentials that are federated by authorized Teams partners. They're also called External users. <br>**See also**: [Anonymous user](#a) |
```

Comment:
Why were these terms removed?

Reply:
@aosolis we removed this content since we wanted to only focus on Meeting Extensibility features, will try to find existing documentation on this one and link here. if not will differ this change for next iteration

Reply:
Ah it's linking to your doc that's being revised. I missed that! :)